ts/features/bonus/cgn/saga/networking/eyca/activation/getEycaActivationStatus.ts

```diff
@@ -0,0 +1,22 @@
+import { call, put } from "redux-saga/effects";
+import { BackendCGN } from "../../../../api/backendCgn";
+import { SagaCallReturnType } from "../../../../../../../types/utils";
+import { cgnEycaActivation } from "../../../../store/actions/eyca/activation";
+import { getActivation } from "./getEycaActivationSaga";
+
+export function* getEycaActivationStatusSaga(
+  getEycaActivation: ReturnType<typeof BackendCGN>["getEycaActivation"]
+) {
+  const activationInfo: SagaCallReturnType<typeof getActivation> = yield call(
+    getActivation,
+    getEycaActivation
+  );
+  if (activationInfo.isLeft()) {
+    yield put(cgnEycaActivation.failure(activationInfo.value));
+    return;
+  }
+
+  if (activationInfo.value === "ERROR") {
```

Comment:
I guess this is a code duplication of an existing handler 
https://github.com/pagopa/io-app/blob/6cf88487d63f80ea111dda3d53bd0db7d6a680f4/ts/features/bonus/cgn/saga/networking/eyca/activation/getEycaActivationSaga.ts#L63
Furthermore this code could generate an **inconsistent state in the store**:
this code dispatches success only if the value is "ERROR" in all other case the store will be `onLoading` despite we received a response from the API